docs/nva-publication-channels.yaml

```diff
@@ -0,0 +1,241 @@
+openapi: 3.0.1
```

Comment:
Is there any special condition that allows the definition of the openapi in a different file in this project? If not, is it possible to start moving the openapi definitions in other projects as well?